ambry-router/src/main/java/com.github.ambry.router/TtlUpdateOperation.java

```diff
@@ -315,14 +324,16 @@ private int getPrecedenceLevel(RouterErrorCode routerErrorCode) {
         return 0;
       case BlobExpired:
         return 1;
-      case AmbryUnavailable:
+      case BlobUpdateNotAllowed:
```

Comment:
It seems in `routerErrorCodeResolutionTest` method, no test case for `BlobUpdateNotAllowed`?

Reply:
thanks, added test